docs/featuretoggles/report.yml

```diff
@@ -0,0 +1,2091 @@
+cms/envs/common.py:
```

Comment:
I get it could. I don't have a very strong opinion on this. Generally speaking, I avoid versioning of automatically-generated assets. If we version this file, then developers will have to remember to update it very often. For instance, it will become outdated as soon as the position of feature toggles in the source code will be modified, as this report includes line numbers. Plus, generating this file requires the installation of code_annotations as well as the [feature_toggle_annotations.yaml](https://github.com/edx/edx-toggles/blob/master/feature_toggle_annotations.yaml) configuration file from the edx-toggles repo, so I bet developers will frequently forget to update the yaml report.

Reply:
I was imagining this being automated so it would be up to date, but that automation clearly doesn't exist yet.
For now, if your code generates and uses this file and it doesn't need to be committed, then feel free to remove it.